### ISSUE TITLE
Switch salt split to not shuffle

### ISSUE THREAD

**Issue by lindsayplatt** [lindsayplatt/salt-modeling-data#5]
Should be as easy as adding `shuffle=False` as an input to `train_test_split()`. I learned this from reading https://towardsdatascience.com/building-rnn-lstm-and-gru-for-time-series-using-pytorch-a46e5b094e7b
